apps/frontend/src/app/3_organisms/ZeroLocForm/CloseCreditLine.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import { SupportedTokens } from '@sovryn/contracts';
+import {
+  Button,
+  FormGroup,
+  Input,
+  InputSize,
+  Paragraph,
+  ParagraphSize,
+  Select,
+  SimpleTable,
+} from '@sovryn/ui';
+
+import { AssetRenderer } from '../../2_molecules/AssetRenderer/AssetRenderer';
+import { translations } from '../../../locales/i18n';
+import { formatValue } from '../../../utils/math';
+import { Row } from './Row';
+import { availableTokens } from './utils';
+
+type CloseCreditLineProps = {
+  collateralValue: string;
+  creditValue: string;
+  availableBalance: string;
+  onSubmit: () => void;
+};
+
+export const CloseCreditLine: FC<CloseCreditLineProps> = ({
+  collateralValue,
+  creditValue,
+  availableBalance,
+  onSubmit,
+}) => {
+  const { t } = useTranslation();
+  const [creditToken, setCreditToken] = useState<SupportedTokens>(
+    SupportedTokens.dllr,
+  );
+
+  const collateralValueRenderer = useCallback(
+    (value: number) =>
+      `${formatValue(value, 6)} ${SupportedTokens.rbtc.toUpperCase()}`,
+    [],
+  );
+
+  const insufficientBalance = useMemo(() => {
+    return Number(availableBalance) - Number(creditValue);
+  }, [availableBalance, creditValue]);
+
+  const hasError = useMemo(
+    () => Number(availableBalance) < Number(creditValue),
+    [creditValue, availableBalance],
+  );
+
+  return (
+    <>
+      <Paragraph
+        className="font-medium"
+        size={ParagraphSize.base}
+        children={t(translations.closeCreditLine.title)}
+        dataLayoutId="close-credit-line"
+      />
```

Comment:
```suggestion
           <Paragraph
        className="font-medium"
        size={ParagraphSize.base}
        children={t(translations.closeCreditLine.title)}
        dataLayoutId="close-credit-line"
        style={ParagraphStyle.tall}
      />
```
Line height should be `1.25rem` according to the design.